release.py

```diff
@@ -512,6 +512,7 @@ def command_release_announcement_email():
 
 jdk8_env = get_jdk(prefs, 8)
 jdk11_env = get_jdk(prefs, 11)
```

Comment:
Can we remove this?

Reply:
Most probably, but I never executed that part of the script.

Reply:
I will push a commit without this line

Reply:
Can you search the file for references to this? If there are none, please remove it.

Reply:
The only reference was at line 604, which I replaced for JDK15

Reply:
Pushed